web/ui/react-app/src/pages/targets/StateIndicator.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { Alert } from 'reactstrap';
+import styles from './StateIndicator.module.css';
+
+export interface StateIndicatorProps {
+  health: string;
+  message?: string;
+}
+
+const getColor = (health: string): string => {
+  switch (health.toLowerCase()) {
+    case 'up':
+      return 'success';
+    case 'down':
+      return 'danger';
+    default:
+      return 'warning';
+  }
+};
+
+const StateIndicator: React.FC<StateIndicatorProps> = ({ health, message }) => {
+  const alertProps = {
+    className: styles.message,
+    color: getColor(health),
+  };
+
+  return <Alert {...alertProps}>{message ? message : health.toUpperCase()}</Alert>;
```

Comment:
Let's just do default-styled `<Badge>` elements in the new UI with `success` / `warning` / `danger` colors, instead of the old `<Alert>` elements with custom styles.